Formula/s/smpeg2.rb

```diff
@@ -1,12 +1,17 @@
 class Smpeg2 < Formula
   desc "SDL MPEG Player Library"
   homepage "https://icculus.org/smpeg/"
-  url "svn://svn.icculus.org/smpeg/tags/release_2_0_0/", revision: "408"
-  head "svn://svn.icculus.org/smpeg/trunk"
+  url "https://github.com/icculus/smpeg/archive/refs/tags/release_2_0_0.tar.gz"
+  sha256 "fdd431bd607efcf0f35789fb3105d7535d4f0e8b46e673e9c0051726e8d1e701"
+  license "LGPL-2.1-or-later"
+  head "https://github.com/icculus/smpeg.git", branch: "main"
 
   livecheck do
-    url "https://svn.icculus.org/smpeg/tags/"
-    regex(%r{href=.*?release[._-]v?(2(?:[._]\d+)+)/}i)
+    url :stable
+    regex(/^release[._-]v?(\d+(?:[._]\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^release[._-]v?(2(?:[._]\d+)+)$/i)
```

If this formula is only for 2.x versions, we should continue to restrict matching to 2.x tags.

Reply:
👍